src/diffusers/schedulers/scheduling_ddim_parallel.py

```diff
@@ -0,0 +1,627 @@
+# Copyright 2023 Stanford University Team and The HuggingFace Team. All rights reserved.
```

Comment:
I think you should also retain the credits here since your team introduced the changes proposed here. 

@patrickvonplaten WDYT?

Reply:
Yeah guess that would be nicer